docs/dev-setup.md

```diff
@@ -174,6 +174,28 @@ refer to
 [the development setup guide of CAPM3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#tilt-for-dev-in-capm3)
 and specially the [Baremetal Operator Integration](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#including-baremetal-operator-and-ip-address-manager)
 
+### Making (virtual) baremetalhost with tilt interface
+
+After Tilt is up it is possible to make baremetalhosts by pressing a button in the Tilt localhost interface (right upper corner). This button runs the content of file 
+```sh
+tools/bmh_test/create_bmh.sh {NAME} {VBMC_PORT}
```

Comment:
```suggestion
tools/bmh_test/create_bmh.sh <NAME> <VBMC_PORT>
```
Usually mandatory parameters are marked with `<>` and optioanl parameters `[]` in documentation.